File: Codechef/VCS/VCS.cpp
#include <iostream>
using namespace std;

int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int N, M, K, first = 0, second = 0;
        cin >> N >> M >> K;
        int A[M], B[K];
        for (int i = 0; i < M; i++)
        {
            cin >> A[i];
        }
        for (int i = 0; i < K; i++)
        {
            cin >> B[i];
        }
        for (int i = 0; i < M; i++)
        {
            for (int j = 0; j < K; j++)
            {
                if (A[i] == B[j])
                {
                    first += 1;
                }
            }
        }
       
        cout << first << " " << N - (M + K - first) << endl;


    }
    return 0;
}

File: Codechef/VCS/VCS.py
T = int(input())
while T:
    T -= 1
    N, M, K = map(int, input().split())
    A = list(map(int, input().split()))
    B = list(map(int, input().split()))
    first, second = 0, 0
    for i in range(1, N + 1):
        if i in A and i in B:
            first += 1
        elif i not in A and i not in B:
            second += 1

    print(str(first), " ", str(second))
